src/main/java/online/book/store/dto/CreateBookRequestDto.java

```diff
@@ -1,15 +1,27 @@
 package online.book.store.dto;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateBookRequestDto {
-    private String title;
+    @NotNull
+    @Size(min = 5, max = 255)
     private String author;
+    @NotNull
+    @Size(min = 5, max = 255)
+    private String title;
+    @NotNull
+    @Pattern(regexp = "\\d{10}|\\d{13}", message = "must be 10 or 13")
```

Comment:
Minor: It would be great to tell user what 10 and 13 are. It is length =)
```suggestion
    @Pattern(regexp = "\\d{10}|\\d{13}", message = "length must be 10 or 13 chars")
```